core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -120,7 +120,7 @@ <h3 class="oppia-donate-h3"><em>Hear from our Oppia community</em></h3>
   }
   .donate-page .oppia-gift-funds-text {
     color: #fff;
-    font-size: 20px;
```

Comment:
Color contrast issues on this text: 
![Screen Shot 2021-04-09 at 3 51 22 PM](https://user-images.githubusercontent.com/35385366/114248081-a41ea200-994b-11eb-97e1-69c99939ffd7.png)

Either darkening background color or increasing size of text will fix the issue. I chose to increase the size of this text

Reply:
Original Size
<img width="375" alt="Screen Shot 2021-04-15 at 1 22 49 PM" src="https://user-images.githubusercontent.com/35385366/114933595-aec6b480-9ded-11eb-8101-839f8db68054.png">


New size: 24px
<img width="378" alt="Screen Shot 2021-04-15 at 1 15 19 PM" src="https://user-images.githubusercontent.com/35385366/114932754-b5086100-9dec-11eb-9e87-5add8eb28d88.png">